src/View/NotAlone.elm

```diff
@@ -81,7 +82,7 @@ renderRevealedCard journeyCardPosition =
     li [ css [ cardStyle, openStyle ] ]
         [ div []
             [ button [ css [ closeJourneyButtonStyle ], onClick (ToggleJourney journeyCardPosition) ]
-                [ img [ css [ height (px 44), margin auto ], src "/Close.svg", alt "Close" ] []
+                [ img [ css [ height (px 44), margin auto ], src "/Close.svg", alt "Close button" ] []
```

Comment:
Is there another instance of close button to change? The emergency popup?

Reply:
I can put this in the keys. Normally a button is read out as such, but it doesn't seem to be here with NVDA

Reply:
Don't mind if you make a key for it or not as long as they match.